argo-cd-apps/overlays/development/delete-applications.yaml

```diff
@@ -121,12 +121,6 @@ $patch: delete
 ---
 apiVersion: argoproj.io/v1alpha1
 kind: ApplicationSet
-metadata:
-  name: kyverno
-$patch: delete
----
-apiVersion: argoproj.io/v1alpha1
```

Comment:
this change is causing kyverno to be deployed in dev overlay. Is this want you want? I am ok with it, just that commit message/PR title is mentioning staging clusters

Reply:
Good catch - development overlays should be a separate PR I think.

Reply:
Fixed in b9234b04.